staging/src/k8s.io/code-generator/cmd/generic-conversion-gen/generators/manual_conversions_tracker.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generators
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"k8s.io/klog"
+
+	"k8s.io/gengo/generator"
+	"k8s.io/gengo/namer"
+	"k8s.io/gengo/types"
+)
+
+// ManualConversionsTracker keeps track of manually defined conversion functions.
```

Comment:
OK, this file / commit is a big improvement in factoring over the existing code, thanks. I am not sure about the location of this file, though-- should it really be under a cmd/ directory? I think it will be shared between multiple commands and therefore should be in some sort of neutral library?